go/libkb/upgrade_instructions.go

```diff
@@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
 	// TODO: non-brew update instructions
 }
 
+func windowsUpgradeInstructions() {
+	serverURI := ProductionServerURI
+	if DefaultRunMode == DevelRunMode {
+		serverURI = DevelServerURI
+	} else if DefaultRunMode == StagingRunMode {
+		serverURI = StagingServerURI
+	}
+
+	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
```

Comment:
is this a direct link to the page, or just the home page? Can you provide direct link to the install page?  Also, we don't need to get this right for staging or devel.